### PR TITLE
docs: add servicebus service-to-service correlation userguide to v2.6 feature docs

### PR DESCRIPTION
**Pull request by stijnmoreels**
We have released Arcus Messaging v1.3 which includes the necessary functionality that is needed to understand the Azure Service Bus service-to-service correlation user guide.
This means we can add the user guide (which already was in preview) to the published v2.6 feature docs.

### CODE REVIEW

File: docs/versioned_docs/version-v2.6.0/02-Guidance/Service-to-service Correlation/index.md [hunk 1]
```diff
@@ -38,4 +38,5 @@ We're working on adding Azure EventHubs message pump to the mix.
 ## How do I add service-to-service correlation to my application components?
 Service-to-service correlation should be added on both the sending and receiving components of your project so the internal Arcus functionality can link the request/response correctly in Application Insights. The following user-guides will go over the sending and receiving side of Web API's and Azure Service Bus resources:
 
-* [Use service-to-service correlation to Web API solutions](use-service-to-service-correlation-in-web-api.md)
\ No newline at end of file
+* [Use service-to-service correlation to Web API solutions](use-service-to-service-correlation-in-web-api.md)
```
Comment (fgheysels) [inline docs/versioned_docs/version-v2.6.0/02-Guidance/Service-to-service Correlation/index.md:41]:
```suggestion
* [Use service-to-service correlation in Web API solutions](use-service-to-service-correlation-in-web-api.md)
```

File: docs/versioned_docs/version-v2.6.0/02-Guidance/Service-to-service Correlation/use-service-to-service-correlation-in-service-bus.md [hunk 2]
```diff
@@ -0,0 +1,351 @@
+---
+title: "Use service-to-service correlation in Azure Service Bus solutions"
+layout: default
+---
+
+# Use service-to-service correlation in Azure Service Bus solutions
+The concept of service-to-service correlation is big and complex and spans multiple Arcus libraries. This user-guide will walk through all the available Arcus features that work together to facilitate service-to-service correlation in your Azure Service Bus solutions. For a general introduction, see the [introduction page on service-to-service correlation](index.md).
+
+To use service-to-service correlation in Azure Service Bus solutions, both the sending and receiving side of the separate components have to be adapted. The correlation will be passed through from one service to another via the Azure Service Bus message application properties which the internal Arcus system will use to link your services together.
```
Comment (fgheysels) [inline docs/versioned_docs/version-v2.6.0/02-Guidance/Service-to-service Correlation/use-service-to-service-correlation-in-service-bus.md:9]:
```suggestion
To use service-to-service correlation in Azure Service Bus solutions, both the sending and receiving side of the separate components have to be adapted. The correlation will be passed through from one service to another via the Azure Service Bus message application properties which Arcus will use to link your services together.
```
Isn't that simpler / easier to read ?

File: docs/versioned_docs/version-v2.6.0/02-Guidance/Service-to-service Correlation/use-service-to-service-correlation-in-service-bus.md [hunk 3]
```diff
@@ -0,0 +1,351 @@
...
+
+The following diagram shows how an Web API calls a Service Bus with correlation tracking:
```
Comment (fgheysels) [inline docs/versioned_docs/version-v2.6.0/02-Guidance/Service-to-service Correlation/use-service-to-service-correlation-in-service-bus.md:11]:
```suggestion
The following diagram shows how a Web API calls a Service Bus with correlation tracking:
```

File: docs/versioned_docs/version-v2.6.0/02-Guidance/Service-to-service Correlation/use-service-to-service-correlation-in-service-bus.md [hunk 4]
```diff
@@ -0,0 +1,351 @@
...
+
+## Service Bus demonstration
+In this user-guide, a fictive API and Service Bus application will be used to represent Service A and Service B of the diagram. Both services will be adapted to use service-to-service correlation.
+
+* **Order API** (Service A): receives an request to order a product, sends the order to process.
+* **Order Worker** (Service B): receives the order request and processes the order.
+
+The user interacts with the **Product API** to order their product. Internally, the **Order API** will contact the **Order Worker** to further process the order request.
+
+> ⚠ Take into account that this sample should only be used for demonstration purposes and does not reflect a fully production-ready implementation. We have chosen to only provide the bare bones of the application so the changes are clear.
```
Comment (fgheysels) [inline docs/versioned_docs/version-v2.6.0/02-Guidance/Service-to-service Correlation/use-service-to-service-correlation-in-service-bus.md:37]:
```suggestion
> ⚠ Take into account that this sample should only be used for demonstration purposes and does not reflect a fully production-ready implementation. We have chosen to only provide the bare bones of the application to be able to focus on the changes required for service-to-service correlation.
```

**Review by fgheysels — APPROVED** [review]
I have some small remarks / improvements.
I know that this is a 'versioned doc', so we already reviewed this, but sometimes you see possible improvements that you didn't saw previously.

**Comment by stijnmoreels** [comment]
> I have some small remarks / improvements. I know that this is a 'versioned doc', so we already reviewed this, but sometimes you see possible improvements that you didn't saw previously.

Yes, no problem. I'll update both the preview and the versioned doc.

File: docs/versioned_docs/version-v2.6.0/02-Guidance/Service-to-service Correlation/use-service-to-service-correlation-in-service-bus.md (hunk 4 above)
Comment (stijnmoreels) [reply docs/versioned_docs/version-v2.6.0/02-Guidance/Service-to-service Correlation/use-service-to-service-correlation-in-service-bus.md]: Rather long for such a short purpose, but ok.

Comment (stijnmoreels) [reply docs/versioned_docs/version-v2.6.0/02-Guidance/Service-to-service Correlation/use-service-to-service-correlation-in-service-bus.md]: https://github.com/arcus-azure/arcus.observability/pull/467/commits/1d7dc7d322057da07d44e7d48fb60b7cb478370e

File: docs/versioned_docs/version-v2.6.0/02-Guidance/Service-to-service Correlation/use-service-to-service-correlation-in-service-bus.md (hunk 3 above)
Comment (stijnmoreels) [reply docs/versioned_docs/version-v2.6.0/02-Guidance/Service-to-service Correlation/use-service-to-service-correlation-in-service-bus.md]: https://github.com/arcus-azure/arcus.observability/pull/467/commits/d3106907ad596c8a06b904ef73e45a95832e8700

File: docs/versioned_docs/version-v2.6.0/02-Guidance/Service-to-service Correlation/use-service-to-service-correlation-in-service-bus.md (hunk 2 above)
Comment (stijnmoreels) [reply docs/versioned_docs/version-v2.6.0/02-Guidance/Service-to-service Correlation/use-service-to-service-correlation-in-service-bus.md]: Yes, that's true. Thx! 👍 
https://github.com/arcus-azure/arcus.observability/pull/467/commits/7afadc0cf3ab390f9fe62845d5cd1ff0abdc8e02

File: docs/versioned_docs/version-v2.6.0/02-Guidance/Service-to-service Correlation/index.md (hunk 1 above)
Comment (stijnmoreels) [reply docs/versioned_docs/version-v2.6.0/02-Guidance/Service-to-service Correlation/index.md]: https://github.com/arcus-azure/arcus.observability/pull/467/commits/7afadc0cf3ab390f9fe62845d5cd1ff0abdc8e02

**Pull request merged**